ui/runs/views.py

```diff
@@ -18,7 +18,8 @@
 from django.shortcuts import render
 from django.urls import reverse
 
-from protzilla.run import Run, get_available_run_names
+from protzilla.run import Run, get_available_run_names 
+from protzilla.run_v2 import delete_run_folder
```

Comment:
zur Klasse run vs. run_v2 sollten wir mal das alte BP fragen, was warum wo verwendet wird und wo dann unsere Änderungen am ehesten rein gehören (ich hätte aber auch am ehesten in run_v2 geschrieben)